JSQMessagesViewController/Views/JSQMessagesInputToolbar.h

```diff
@@ -83,6 +83,17 @@
 @property (assign, nonatomic) BOOL sendButtonOnRight;
 
 /**
+ *  Specify if the send button might be enabled.
+ *
+ *  @discussion Normally the send button will be enabled if the `textView` contains text. This
+ *  property overrides this behaviour, for example to disable the send button, if the internet
+ *  connection is lost.
+ *
+ *  @see `updateSendButtonEnabledState`.
+ */
+@property (assign, nonatomic) BOOL sendButtonEnabled;
```

Comment:
I would vote for `sendButtonDisabled`. It's intuitive for developers to control the send button. And I propose to deprecate `toggleSendButtonEnabled`. Instead, let property setter to handle all necessary internal changes.